src/app-layout/visual-refresh-toolbar/skeleton/styles.scss

```diff
@@ -72,34 +72,45 @@
 }
 
 .navigation,
-.tools {
+.tools,
+.global-tools {
   grid-row: 1 / -1;
   grid-column: 1 / -1;
   background: awsui.$color-background-container-content;
-  z-index: constants.$drawer-z-index;
   opacity: 1;
   @include mobile-only {
     inline-size: 100%;
-    z-index: constants.$drawer-z-index-mobile;
   }
 }
 
 .navigation {
+  z-index: constants.$drawer-z-index;
+
   @include desktop-only {
     grid-area: navigation;
     inline-size: var(#{custom-props.$navigationWidth});
     border-inline-end: awsui.$border-divider-section-width solid awsui.$color-border-layout;
   }
+
+  @include mobile-only {
+    z-index: constants.$drawer-z-index-mobile;
+  }
 }
 
 .tools {
   @include desktop-only {
     grid-area: tools;
-    inline-size: var(#{custom-props.$toolsWidth});
     border-inline-start: awsui.$border-divider-section-width solid awsui.$color-border-layout;
   }
 }
 
+.global-tools {
+  @include desktop-only {
+    display: flex;
+    grid-area: global-tools;
```

Comment:
**Before merge:**

Why do we not assign the grid area on mobile? Leaving the area unassigned may cause problems

Reply:
It's the way it works for navigation and tools panels now, everything is fine on mobile view. I don't see any reason to implement it differently

Reply:
got it, ok